lib/internal/streams/destroy.js

```diff
@@ -8,7 +8,10 @@ function destroy(err, cb) {
     this._writableState.destroyed;
```

Comment:
IMHO If the `cb` should stay undocumented don't put it in the signature, extract it from `arguments`.
My IDE (WebStorm) will intellisense it's presence.